hangups/client.py

```diff
@@ -776,6 +776,30 @@ def setchatname(self, conversation_id, name):
             raise exceptions.NetworkError()
 
     @asyncio.coroutine
+    def setchatnotificationlevel(self, conversation_id, level):
```

Comment:
This method should be named `setconversationnotificationlevel` to match the API it uses.